plugin-ci/orb.yml

```
@@ -104,11 +104,22 @@ jobs:
       - checkout
       - *restore_cache
       - run: make dist
+      - run:
+          name: "Generating release notes"
+          command: |
+            echo "### Supported Mattermost Server Versions: $(cat plugin.json | jq .min_server_version -r)+"  >> release-notes.md
```

I would like to stay consistent with the existing structure:
```suggestion
            echo "Supported Mattermost Server Versions: **$(cat plugin.json | jq .min_server_version -r)+**"  >> release-notes.md
```